index.html

```diff
@@ -29,19 +29,21 @@
                             17 10H1Z" fill="#6070FF" />
                 </svg>
                 <ul class="menu show-if-large hidden-if-small">
-                    <li class="p-for-textbox"><span class="text">Portfolio</span></li>
-                    <li class="p-for-textbox"><span class="text">About</span></li>
-                    <li class="p-for-textbox"><span class="text">Contact</span></li>
+                    <li class="p-for-textbox"><a href="#"><span class="text">Portfolio</span></a></li>
+                    <li class="p-for-textbox"><a href="#"><span class="text">About</span></a></li>
+                    <li class="p-for-textbox"><a href="#"><span class="text">Contact</span></a></li>
```

Comment:
[OPTIONAL] Please add the section's reference on the _href_ of these `a` tags to improve the navigation of your website!

Reply:
Fixed.